osbuild-composer.spec

```diff
@@ -77,6 +77,9 @@ Obsoletes: lorax-composer < 34.3
 Obsoletes: golang-github-osbuild-composer < %{version}-%{release}
 Provides:  golang-github-osbuild-composer = %{version}-%{release}
 
+# remove in F35
```

Comment:
As we use the same spec file for every distro, I suggest `remove when F34 is EOL`. I think this is more obvious to future readers.

Reply:
Makes sense. Thanks!